dts/bindings/sensor/st,stm32-temp.yaml

```diff
@@ -39,7 +39,7 @@ properties:
 
   v25:
     type: int
-    default: 760
+    required: true
```

Comment:
Please move this in a different commit, along with potential impacts reflected in dtsi or a comment explaining that there is strangely no impact.